pkg/migrations/op_common_test.go

```diff
@@ -296,6 +296,27 @@ func indexExists(t *testing.T, db *sql.DB, schema, table, index string) bool {
 	return exists
 }
 
+func CheckIndexDefinition(t *testing.T, db *sql.DB, schema, table, index, expectedDefinition string) {
```

Comment:
We sould use this also in the `create partial index` test added in a previous PR that currently just checks for index existence.